server/game/sprites/items.js

```diff
@@ -139,6 +141,79 @@ class Cow extends Ingredient {
     this.defaultFlipY = true
     this.xPad = 30
     this.yPad = 30
+
+    // Randomly change direction after every X milliseconds
+    this.timeFromLastMove = 0
+    this.newMoveDelay = 4000
+
+    // If moveLeft is true, the cow will move left, otherwise move right
+    this.moveLeft = true
+
+    // Reference to cloner that the cow came from
+    // Mainly used to keep track of the cows that have been created by the cloner
+    // Not the best approach, so probably want to improve on this later
+    this.clonerSource = null
+  }
+
+  removeEvents() {
+    // TODO: May need to rename this function to be something more like clean up or something
+    super.removeEvents()
+    // Need to decrement cow count when the cow is removed from the game, this way new cows
+    // can be created
+    this.clonerSource.subtractCow()
+    this.clonerSource = null
+  }
+
+  setClonerSource(cloner) {
+    this.clonerSource = cloner
+  }
+
+  shootUp() {
+    // Logic for shooting a cow from the cloner
+    // Randomly pick the x/y velocities so the cows fall in different directions
+    this.body.angularVelocity = 500
+    this.setVelocityY(random(-600, -1000))
```

Comment:
Phaser.Math.Between generates a random number between that range. No need to change, but it's nice that Phaser offers some of these helping functions: https://photonstorm.github.io/phaser3-docs/Phaser.Math.html

Reply:
Ah nice. Didn't know about that. I did see that when I was looking at your super elder code (I was looking at it to see how you did the timed projectiles, lol)

Reply:
Updated.